src/posix.mak

```diff
@@ -463,7 +463,7 @@ cxx-unittest: $(GENERATED)/build
 
 ######################################################
 
-zip: $(GENERATED)/build
+zip: detab tolf $(GENERATED)/build
```

Comment:
```suggestion
zip: $(GENERATED)/build
```
`detab` and `tolf` are not defined in this makefile. Of course, would be better if we delete the `zip` target altogether, but let's leave this for another PR.

Reply:
thanks for finding that